translations/translation_ru.xml

```diff
@@ -205,8 +205,8 @@
 		<text name="COURSEPLAY_BGA_IS_FULL" 							 text=": силосная яма заполнена!" />
 		<text name="COURSEPLAY_FARM_SILO_NO_FILLTYPE" 					 text="В этом бункере нет такого продукта!" />
 		<text name="COURSEPLAY_FARM_SILO_FILL_TYPE" 					 text="Забрать из бункера:" />
-		<text name="COURSEPLAY_FARM_SILO_IS_EMPTY" 						 text="В бункере не осталось выбранного продукта, увы...! Waiting for more." />  <!-- TODO: TRANSLATION! Translate new added text. -->
```

Comment:
the last text added on this line is to notify the user that is waiting for new crops to be filled into the silo.
the original english text is:
> Farm silo is empty! Waiting for more.

But since I don't know what it says in Russian I cant tell if it's needed there or not :)